legend-engine-ide-lsp-server/src/test/java/org/finos/legend/engine/ide/lsp/server/integration/TestLegendLanguageServerReplIntegration.java

```diff
@@ -39,13 +43,26 @@ void testReplStartWithGivenClasspath(@TempDir Path dir) throws Exception
     {
         String classpath = extension.futureGet(extension.getServer().getLegendLanguageService().replClasspath());
 
+        if (OS.WINDOWS.isCurrentOs())
+        {
+            classpath = classpath.replace(File.separator, "/");
+        }
+        if (classpath.contains(" "))
+        {
+            classpath = Arrays.stream(classpath.split(File.pathSeparator)).map(x -> String.format("\"%s\"", x)).collect(Collectors.joining(File.pathSeparator));
+        }
+
+        Path classpathFile = Files.writeString(dir.resolve("classpath"), classpath);
+
         ProcessBuilder processBuilder = new ProcessBuilder(
                 System.getProperty("java.home") + File.separator + "bin" + File.separator + "java",
+                "-cp",
+                "@" + classpathFile.toString(),
```

Comment:
In some CICD environments, the classpath is too long and leads to failures.  This change aims to reduce the length of the process arguments by using a file for classpath.